sirepo/server.py

```diff
@@ -657,7 +658,8 @@ def init(uwsgi=None, use_reloader=False, is_server=False):
     _app = flask.Flask(
         __name__,
         static_folder=None,
-        template_folder=str(simulation_db.STATIC_FOLDER),
+        # TODO(e-carlin): I don't think we use templates. Talk with rn
```

Comment:
@robnagler what was this line for? We don't use flask templates right?

Reply:
Ah, right. Good fix, but the commit should have documented it. That's a problem with PRs. I'm only looking at that comment.

Reply:
1b7d57ae